CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+##Supported Release 1.4.1
```

Comment:
Hash-section syntax is supposed to have a space between the hashes and the text.